### PR TITLE
Merge relational/inheritance into inheritance page

### PR DESCRIPTION
**Pull request by roji**
Part of #1669

### CODE REVIEW

File: entity-framework/core/modeling/inheritance.md [hunk 1]
```diff
@@ -8,28 +8,49 @@ uid: core/modeling/inheritance
...
+[!code-csharp[Main](../../../samples/core/Modeling/Conventions/InheritanceDbSets.cs?name=InheritanceDbSets&highlight=3-4)]
 
-> [!NOTE]
-> You can use `.HasBaseType((Type)null)` to remove an entity type from the hierarchy.
+This model be mapped to the following database schema (note the implicitly-created *Discriminator* column, which identifies which type of *Blog* is stored in each row):
+
+![image](_static/inheritance-tph-data.png)
+
+>[!NOTE]
+> Database columns are automatically made nullable as necessary when using TPH mapping. For example, the *RssUrl* column is nullable because regular *Blog* instances do not have that property.
```
Comment (roji) [inline entity-framework/core/modeling/inheritance.md]: Is this worth talking about this? Seems like an under-the-hood implementation detail, no?

Comment (smitpatel) [reply entity-framework/core/modeling/inheritance.md]: No preference but seen quite a few customers looking at migration and complaining that column is nullable even though they set it as Required.

Comment (roji) [reply entity-framework/core/modeling/inheritance.md]: OK, if this is a frequent source of confusion we can leave it in.

File: entity-framework/core/modeling/inheritance.md [hunk 2]
```diff
@@ -8,28 +8,49 @@ uid: core/modeling/inheritance
...
 
-## Conventions
+At the moment, EF Core only supports the table-per-hierarchy (TPH) pattern. TPH uses a single table to store the data for all types in the hierarchy, and a discriminator column is used to identify which type each row represents.
 
-By default, it is up to the database provider to determine how inheritance will be represented in the database. See [Inheritance (Relational Database)](relational/inheritance.md) for how this is handled with a relational database provider.
+> [!NOTE]
+> The table-per-type (TPT) and table-per-concrete-type (TPC), which are supported by EF6, are not yet supported by EF Core. TPT is a major feature planned for EF Core 5.0.
 
-EF will only setup inheritance if two or more inherited types are explicitly included in the model. EF will not scan for base or derived types that were not otherwise included in the model. You can include types in the model by exposing a *DbSet\<TEntity>* for each type in the inheritance hierarchy.
+## Included hierarchy types
```
Comment (AndriySvyryd) [inline entity-framework/core/modeling/inheritance.md]: Default hierarchy mapping?

Comment (roji) [reply entity-framework/core/modeling/inheritance.md]: Well, this section does discuss how to use the fluent API to include types which aren't included "by default"...

How about `Hierarchy type mapping`? Am open to other ideas too.

Comment (roji) [reply entity-framework/core/modeling/inheritance.md]: Or `Hierarchy entity type mapping`?

Comment (AndriySvyryd) [reply entity-framework/core/modeling/inheritance.md]: Entity type hierarchy mapping?

Comment (roji) [reply entity-framework/core/modeling/inheritance.md]: Sold.

**Pull request merged**